2.0/Box/glTF-pbrSpecularGlossiness/Box.gltf

```diff
@@ -112,8 +112,26 @@
                     0.0,
                     0.0,
                     1.0
-                ]
-            }
+                ],
+                "metallicFactor": 0.0
+            },
+            "extensions": {
+                "KHR_materials_pbrSpecularGlossiness": {
+                    "diffuseFactor": [
+                        0.800000011920929,
+                        0.0,
+                        0.0,
+                        1.0
+                    ],
+                    "specularFactor": [
+                        0.20000000298023225,
```

Comment:
> Do we want SG to look different from MR in these models?

- Since these are sample models, we can expect that they will be used as "proper" examples. So results must be predictable.
- If the goal of using both materials in one asset is to achieve better compatibility/performance, they should look "similar".
- We should provide additional samples to highlight differences between MR and SG.